string model_name
uint32[] epochs
---
float64[] probs